### ISSUE TITLE
Source maps

### ISSUE THREAD

**Issue by pablorc** [mobilexag/plugin-sass#20]
Hi!

Before all, great plugin! Is in your planning, or exists a way, to create sourcemaps for the generated CSS? I'm migrating a system into JSPM and extending the scss file would be a pain with all these minification.

**Comment by screendriver** [comment]
Hi @pablorc,

at runtime there is no minification. It only occurs on bundling time. Bundling is meant for production (so everything gets minified and uglified) where runtime injection is for development time.

I could enable source map support for runtime injection if you want?

**Comment by henrymdev** [comment]
Hi @screendriver 

I like the plugin as well, could you please add the source map support for runtime injection.

**Comment by screendriver** [comment]
I played around a little bit but I don't get source maps running when they are included inline in the html file. It works if you use external loaded `.css` files with included source maps but not when you include the base64 maps directly into the `<style>` tags.

**Comment by yyaari** [comment]
Indeed very good plugin. thanks!
I do not entirely agree that bundling is only for production, for medium/large applications it is impossible to use systemjs un-bundled as the app takes forever to load on every refresh. 
instead we are using increment bundling which works way better, however not being able to use source maps with our sass files is very limiting.
Is there anyway to use the plugin with source maps (css is not miniified in our dev flow)?

Thanks

**Comment by screendriver** [comment]
That's true. In large projects everything is too slow. I would like to add source maps, but unfortunately I don't know how. If anyone would find a way to do that, pull requests are welcome.
